logspout/logspout.go

```diff
@@ -65,32 +70,42 @@ func syslogStreamer(target Target, types []string, logstream chan *Log) {
 			continue
 		}
 		tag, pid, data := getLogParts(logline)
-		var conn net.Conn
+
+		// HACK: Go's syslog package hardcodes the log format, so let's send our own message
+		data = fmt.Sprintf("%s %s[%s]: %s",
+			time.Now().Format(getopt("DATETIME_FORMAT", dtime.DeisDatetimeFormat)),
+			tag,
+			pid,
+			data)
+
 		if strings.EqualFold(target.Protocol, "tcp") {
 			addr, err := net.ResolveTCPAddr("tcp", target.Addr)
 			assert(err, "syslog")
-			tcpconn, err := net.DialTCP("tcp", nil, addr)
+			conn, err := net.DialTCP("tcp", nil, addr)
+			assert(err, "syslog")
+			assert(conn.SetWriteBuffer(MAX_TCP_MSG_BYTES), "syslog")
```

Comment:
In this case, "assert" is actually an error handing function defined here:

https://github.com/krancour/deis/blob/a0da307f4fec1492b5883e823bd01b14d4d6a232/logspout/logspout.go#L36

(I didn't write that.)